visualization/app/.env

```diff
@@ -0,0 +1,10 @@
+REACT_APP_AMPLIFY_INCLUDE_GREETING=false
+REACT_APP_AMPLIFY_REGION=us-east-1
+REACT_APP_AMPLIFY_USER_POOL_ID=us-east-1_nNB31bPdm
+REACT_APP_AMPLIFY_USER_POOL_WEB_CLIENT_ID=4iso70ukf2iloror97ll7b9vq7
+REACT_APP_BUFFER_RING_SIZE=250
+REACT_APP_UPDATE_BATCH_SIZE=25
+REACT_APP_UPDATE_TIMEOUT=250
+REACT_APP_WEBSOCKET_API_STUB=false
+REACT_APP_WEBSOCKET_API_URL=wss://f5t62g9095.execute-api.us-east-1.amazonaws.com/Prod
```

Comment:
For the region, user_pool and client IDs, and websocket URL, should we (can we?) be pulling these from CFN?

Reply:
I imagine we could find a way to do that as part of a build process. I'll look into that if we decide to deploy this to S3 instead of just running it locally.